tensorflow/core/kernels/mkl_conv_grad_input_ops.cc

```diff
@@ -43,10 +44,23 @@ limitations under the License.
 #include "mkl_dnn.h"
 #include "mkl_dnn_types.h"
 
+#ifdef INTEL_MKL_DNN
```

Comment:
I see you added this macro, but the way you set things up, when MKL is enabled MKL-DNN will also be enabled.
Is there a case where we want MKL but not MKL-DNN?

Reply:
The intention is to support both for a while since MKL-DNN work is still going on. Using --config=mkl will enable MKL only. Using --config=mkl and --copt="-DINTEL_MKL_DNN" will enable MKL-DNN.